config/locales/js/nl.yml

```
@@ -324,3 +324,6 @@ nl:
       dark: Donker
       system: Systeem
       theme: Stijl
+    feedbacks:
+      submission:
+        submit: Dien deze oplossing in
```

```suggestion
        submit: Deze oplossing indienen
```
This is more consistent with the button next to it